libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -255,8 +205,8 @@ void AC_AutoTune::send_step_string()
     case UPDATE_GAINS:
         gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: UPDATING_GAINS");
         return;
-    case TWITCHING:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: TWITCHING");
+    case TESTING:
```

Comment:
I'm not a fan of this name change. Testing for me is the pilot testing afterwards.

Reply:
This change was in the subclass split PR which @rmackay9 reviewed.  I didn't receive any comments from him on this.  I went through significant testing of the subclass split PR to show that the multi autotune operated nearly identical to autotune before the split.